src/java/com/zimbra/qa/selenium/projects/ajax/core/AjaxCore.java

```diff
@@ -138,8 +139,28 @@ public void coreBeforeSuite() throws HarnessException, IOException, InterruptedE
 	}
 
 	@BeforeClass(groups = { "always" })
-	public void coreBeforeClass() throws HarnessException {
+	public void coreBeforeClass() throws HarnessException, IOException {
 		logger.info("BeforeClass: start");
+		logger.info("Class name: " + this.getClass());
+		ArrayList<String> zimletList= null;
+		if (this.getClass().getName().contains("com.zimbra.qa.selenium.projects.ajax.tests.network.chat")) {
```

Comment:
1. Lets use ajax.tests.zextras.chat for FOSS and ajax.tests.network.zextras.ngmodule for NETWORK.
2. Chat, drive are open source feature so probably needs to move to zm-selenium under zextras directory?